src/components/date-picker/date-picker.ts

```diff
@@ -259,7 +259,7 @@ class BXDatePicker extends LitElement {
   calendar: FlatpickrInstance | null = null;
 
   /**
-   * The date format to use. Corresponds to `date-format` attribute.
+   * The frdate format to let Flatpickr use. Corresponds to `date-format` attribute.
```

Comment:
```suggestion
   * The date format to let Flatpickr use. Corresponds to `date-format` attribute.
```

Unless `frdate` is a specific kind of format?

Reply:
My apologies 🙇 I think it simply was a typo.